tests/backup_role/test_backup.yml

```diff
@@ -0,0 +1,418 @@
+---
+- name: Test ansible-freeipa backup role
+  hosts: ipaserver
+  become: yes
+  gather_facts: yes
+
+  tasks:
+  - name: Get current directory.
```

Comment:
What is `pwd` for? Later on `"{{ lookup('env', 'PWD') }}"` is used.

Reply:
fixed.